dynamicpage.js

```diff
@@ -0,0 +1,250 @@
+            <div class="speaker-content">
+              <h2>${data[5].name}</h2>
+              <p class="about-title2">${data[5].title}</p>
+              <hr>
+              <p class="about-content">${data[5].description}</p>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+
+
+
+</div>
+
+</div>
+
+
+
+<div class="more-desktop">
+<div class="container text-enter about-main-programs">
+
+  <div class="speaker-content1">
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[0].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[0].name}</h2>
+        <p class="about-title2">${data[0].title}</p>
+        <hr>
+        <p class="about-content">${data[0].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[1].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[1].name}</h2>
+        <p class="about-title2">${data[1].title} </p>
+        <hr>
+        <p class="about-content">${data[1].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[2].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[2].name}</h2>
+        <p class="about-title2">${data[2].title}</p>
+        <hr>
+        <p class="about-content">${data[2].description}</p>
+      </div>
+
+    </div>
+  </div>
+
+  <div class="speaker-content2">
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[3].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[3].name}</h2>
+        <p class="about-title2">${data[3].title}</p>
+        <hr>
+        <p class="about-content">${data[3].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[4].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[4].name}</h2>
+        <p class="about-title2">${data[4].title}</p>
+        <hr>
+        <p class="about-content">${data[4].description}</p>
+      </div>
+
+    </div>
+
+    <div class="column2 ">
+      <i><img class="speaker" src="${data[5].image}" alt=""></i>
+      <div class="speaker-content">
+        <h2>${data[5].name}</h2>
+        <p class="about-title2">${data[5].title}</p>
+        <hr>
+        <p class="about-content">${data[5].description}</p>
+      </div>
+
+    </div>
+  </div>`;
+
+const speakersContainer = document.querySelectorAll('.speakers');
+for (let i = 0; i < data.length; i += 1) {
+  const speaker = data[i];
+  const speakerDiv = document.createElement('div');
+  speakerDiv.classList.add('column2');
+  speakerDiv.innerHTML = `
+    <i><img class="speaker" src="${speaker.image}" alt=""></i>
+    <div class="speaker-content">
+      <h2>${speaker.name}</h2>
+      <p class="about-title2">${speaker.title}</p>
+      <hr>
```

Comment:
- [ ] kindly consider giving this a width so as it looks exactly like the one in the provided design 
- your design 👇 
<img width="1133" alt="Screen Shot 2023-04-20 at 20 19 36" src="https://user-images.githubusercontent.com/60146030/233453869-5975e999-2ec7-4ccf-800b-1ef316672717.png">


- provided one 
<img width="967" alt="Screen Shot 2023-04-20 at 20 18 14" src="https://user-images.githubusercontent.com/60146030/233453988-cc6c2c64-9441-4230-9cab-83b0beeb6c3f.png">

Reply:
![image](https://user-images.githubusercontent.com/111823606/233483339-c6681ee8-f212-44ef-8d16-7461c4349c82.png)
Did it, thanks

Reply:
![image](https://user-images.githubusercontent.com/111823606/233483510-079254b4-ea2b-49f6-ad8b-49150410bef7.png)
Did it!!.